features/ubuntu_pro.feature

```diff
@@ -179,10 +179,10 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
             """
 
         Examples: ubuntu release
-           | release | fips-s   | cc-eal-s | cis-s    | infra-pkg | apps-pkg |
-           | xenial  | n/a      | disabled | disabled | libkrad0  | jq       |
-           | bionic  | disabled | n/a      | disabled | libkrad0  | bundler  |
-           | focal   | disabled | n/a      | disabled | hello     | ant      |
+           | release | fips-s   | cc-eal-s | cis-s    | infra-pkg | apps-pkg | livepatch |
+           | xenial  | n/a      | disabled | disabled | libkrad0  | jq       | enabled   |
+           | bionic  | disabled | n/a      | disabled | libkrad0  | bundler  | n/a       |
+           | focal   | n/a      | n/a      | disabled | hello     | ant      | n/a       |
```

Comment:
Based on https://wiki.ubuntu.com/Kernel/Livepatch I would expect focal fips to be enabled on azure pro as well.

That would be a contract change, not on our end, so not blocking

Reply:
We don't allow focal on Azure in uaclient at the moment because we avoid [overwriting the cloud-optimized kernel for additionalPackages on anything other than bionic](https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/entitlements/fips.py#L227-L228)

Reply:
Ahh I got my wires crossed. This is truly a contract change/gap that needs addressing for azure pro and azure generic contracts. Bug to file.

Reply:
I also got my wires crossed haha, this comment should say "focal livepatch"